seqr/views/utils/variant_utils.py

```diff
@@ -239,6 +239,12 @@ def get_variant_key(xpos=None, ref=None, alt=None, genomeVersion=None, **kwargs)
     return '{}-{}-{}_{}'.format(xpos, ref, alt, genomeVersion)
 
 
+def _requires_transcript_metadata(variant):
+    if isinstance(variant, list):
+        return _requires_transcript_metadata(variant[0])
```

Comment:
Can you just call `_requires_transcript_metadata` with variant[0] in the first place?

Reply:
sometimes `variant ` is a list of objects and sometimes its a single object so without the instance check `variant[0]` would throw an error